tests/rptest/tests/compatibility/arroyo_test.py

```diff
@@ -42,7 +42,7 @@ def test_arroyo_test_suite(self):
                     combine_stderr=True,
                     timeout_sec=120):
                 self.logger.info(line)
-                if 'FAILED' in line:
+                if 'FAILED' in line or 'FAILURE' in line:
```

Comment:
nit: is this change strictly needed?

Reply:
yup your right, if FAILURE is printed FAILED will have also been, removing

Reply:
Did you mean to remove the FAILURE part here?